sections/futures/TradingHistory/TradesHistoryTable.tsx

```diff
@@ -239,6 +239,7 @@ const StyledTable = styled(Table)<{ mobile?: boolean }>`
 	.table-body-row {
 		${TableAlignment}
 		padding: 0;
+		height: 55px;
```

Comment:
Looks like this widened the row, but the content is no longer vertically centered:
<img width="295" alt="image" src="https://user-images.githubusercontent.com/10401554/186522691-3b5a8495-34e7-4309-b2b1-53618957f48a.png">